.gitignore

```diff
@@ -0,0 +1,2 @@
+# Obsidian files
+.obsidian
```

Comment:
You can add an empty line in the end of your files to avoid having this
![image](https://user-images.githubusercontent.com/28428558/213209586-afa314e7-aef0-4ba0-b19c-a05e2aceb5f4.png)

Reply:
Done